src/main/resources/application.yaml

```diff
@@ -30,4 +30,3 @@ idam:
     id: 'bsp'
     secret: '123456'
     redirect_uri: 'http://localhost/receiver'
-  users:
```

Comment:
I think some docs need adding on how to set the users for local? as .env didn't work as expected

Reply:
not sure what you mean.